### PR TITLE
Added saving server configs to server database as well as locally to the bot

### PR DESCRIPTION
**Pull request by OmerSabic**
**Please describe the changes this PR makes and why it should be merged, as well as any and all proof of successful tests:**
Instead of having the code interact with enmap directly, I made a class that imitates all the functions of enmap but before setting anything to push it to the API. It also gets all the current configs from the API and stores them locally. This will need to be changed if we decide to shard the bot.

**Semantic versioning classification:**

- [x] This PR changes the framework's interface (methods or parameters added)
  - [ ] This PR includes breaking changes (methods removed or renamed, parameters moved or removed)
- [ ] This PR **only** includes non-code changes, like changes to README, etc.

### CODE REVIEW

File: events/messageCreate.js [hunk 1]
```diff
@@ -18,7 +18,7 @@ module.exports = async (client, message) => {
   const settings = message.settings = getSettings(message.guild);
 
   //Ignore any channel which isn't #testing, EXCEPT DMs
-  if (message.guild === null || message.guild.id !== "953924192259170335") return;
+  // if (message.guild === null || message.guild.id !== "953924192259170335") return;
```
Comment (swatarianess) [inline events/messageCreate.js:21]: Delete them if it isn't being used anymore 😉

File: modules/functions.js [hunk 2]
```diff
@@ -1,6 +1,6 @@
 const logger = require("./logger.js");
 const config = require("../config.js");
-const { settings } = require("./settings.js");
+const settings = require("./settings.js");
 // Let's start by getting some useful functions that we'll use throughout
 // the bot, like logs and elevation features.
```
Comment (swatarianess) [inline modules/functions.js:5]: Not really needed.

**Review by swatarianess — CHANGES_REQUESTED** [review]

File: events/messageCreate.js (hunk 1 above)
Comment (OmerSabic) [reply events/messageCreate.js:21]: I think that was already there, will fix

File: modules/functions.js (hunk 2 above)
Comment (swatarianess) [reply modules/functions.js:5]: its fine, delete it 🙈

**Review by swatarianess — APPROVED** [review]

**Pull request merged**